modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -326,13 +326,45 @@
   [_]
   #{"INFORMATION_SCHEMA"})
 
+(defn- connecting-with-local-keypair?
+  [{:keys [account password private-key-value private_key_file user]}]
+  (and (nil? password)
+       (nil? private_key_file)
+       (some? private-key-value)
+       (some? account)
+       (some? user)))
+
+(let [ByteArray (type (byte-array []))]
```

Comment:
I did it this way because

```clojure
(type (byte-array []))
;;=> [B
```

and `[B` is not readable. Is there a better way to do this?